### DISCUSSION TITLE
第三节课，MINIMUM_VALUE定义不太理解，为什么是100美元，而不是100ETH ?

### DISCUSSION THREAD

**Discussion by skillstar** [smartcontractkit/Web3_tutorial_Chinese#37] [Q&A] [answered]
第三节课，MINIMUM_VALUE定义不太理解，为什么是100美元，而不是100ETH ?
unit256 MINIMUM_VALUE = 100 * 10 ** 18; //最小单位100美元

不是 ETH是 10的18次方，美元是10的8次方吗？
美元不是用预言机动态换算的吗？ 这里怎么写死了？

**Comment by golds3** [comment]
预言机得到的值 是1 eth 等于多少美元，要求最少要100美元，就是要计算用户付的eth值多少美元，即amount*预言机的返回，而对于输入的amount它的单位是wei，所以有10**18这个换算，因此为了正确比较，我们设定的MINIMUM_VALUE也应该乘上10**18
至于美元的10**8，其实应该说写在预言机返回那里
    function getChainlinkDataFeedLatestAnswer() public view returns (int) {
        // prettier-ignore
        (
            /* uint80 roundID */,
            int answer,
            /*uint startedAt*/,
            /*uint timeStamp*/,
            /*uint80 answeredInRound*/
        ) = dataFeed.latestRoundData();
        // dataFeed.latestRoundData() 返回还不是usd的价格，需要除以（usd的精度）10 ** 8才是USD
        return answer/10**8;
    }
而老师把这个动作放在了后面，所以初看起来说有点绕的

**Comment by QingyangKong** [comment, marked as answer]
请查看这个回答：https://github.com/smartcontractkit/Web3_tutorial_Chinese/discussions/10

把加密货币的最小单位默认为 wei，就很好了解了
